### ISSUE TITLE
STEP dataset questions

### ISSUE THREAD

**Issue by lxtGH** [google-research/deeplab2#59]
Hi! Thanks for sharing the code. Is there any validation set for STEP MOTS? Or any suggestions for making the subset from the train set.

**Comment by markweberdev** [comment]
Hi,

Please note that STEP and MOTS are two different things.

- STEP (Segmenting and Tracking Every Pixel) contains labels for every pixel that contain semantic class and potentially tracking labels.
- MOTS (Multi-Object Tracking and Segmentation) does not contain labels for every pixel, but only for certain object classes.

We are offering KITTI-STEP and MOTChallenge-STEP labels.

Please find the guides here:
- [KITTI-STEP](https://github.com/google-research/deeplab2/blob/main/g3doc/setup/kitti_step.md#prepare-kitti-step-for-training-and-evaluation). This includes also how to split the training and validation set.
- [MOTChallenge-STEP](https://github.com/google-research/deeplab2/blob/main/g3doc/setup/motchallenge_step.md). This does not have an official validation split due to the low number of sequences. Both training sequences do not contain the full label set, hence validation is difficult. Instead one could split each training sequence in two parts and train on one part while evaluating on the other.

Please let me know if that answers your question.

**Comment by lxtGH** [comment]
@markweberdev Hi! Thanks for your reply. I wonder "Instead one could split each training sequence in two parts and train on one part while evaluating on the other."  If I do that, what is  the gap between the such setting with test set since we have uploaded the results to the server with the limited times ?

**Comment by markweberdev** [comment]
That I simply don't know, sorry. Please feel free to let us know if that's similar in case you try it. Then you should be able to compare it with the test set.

**Comment by lxtGH** [comment]
@markweberdev Hi！I down the your STEP model and perform evaluation.  However the results are nearly 0 . Could you share your comand for evaluation the STEP models?
![image](https://user-images.githubusercontent.com/17202268/135745856-2df2aec6-57a6-4556-9abb-9d77b81458f1.png)

**Comment by markweberdev** [comment]
@lxtGH The command I am using is the following:

`python trainer/train.py --config_file="./configs/kitti/panoptic_deeplab/resnet50_os32.textproto" --num_gpus=1 --mode=eval --model_dir="/**retracted**/models/deeplab2/kitti/"
`

If you prepare the KITTI dataset correctly and download the pretrained models, you will get the scores we report.

**Comment by aquariusjay** [comment]
Closing the issue, since it is not active for a while.
Please feel free to reopen it if needed.
